### ISSUE TITLE
[FEAT] Nested flashcards.

### ISSUE THREAD

**Issue by Jarartur** [st3v3nmw/obsidian-spaced-repetition#606]
**Is your feature request related to a problem? Please describe.**
I'm used to a concept-descriptor framework from RemNote and I think it would be beneficial to sometimes use it in obsidian. You create notes in the following way:

Concept (ex. Signal processing methods):
?
    - descriptor 1 (ex. ECG:: electrocardiogram)
    - descriptor 2 (ex. EGG:: electrogastrogram)

**Describe the solution you'd like**
This way it would generate 3 flashcards:
1. Multiline flashcard for Concept ? desc1 & desc2
2. Simple flashcard for descriptor 1
3. Simple flashcard for descriptor 2

**Additional context**
This way we can create notes with a flow to them and not repeat ourselves. Currently to achieve it I would need to type:

```
Signal processing methods:
?
- ECG
- EGG

ECG meaning >> electrocardiogram
EGG meaning >> electrogastrogram
```

**Comment by ronzulu** [comment]
Hey there

I'm not familiar with RemNote. Is there just the single level of nesting?

**Comment by Jarartur** [comment]
No, in remnote there is no thing like files/folders so everything is kind of in a parent-child structure. This means that I can go indefinitely down with my flashcards. I do not necessarily think it needs to be cloned 1:1 in obsidian but nesting even a few flashcards of different types can be beneficial in adding more context to the flashcards while keeping them atomic.
